pkg/common/utils_test.go

```diff
@@ -1220,6 +1228,29 @@ func TestCodeForError(t *testing.T) {
 			inputErr: &googleapi.Error{Code: http.StatusBadRequest, Message: "User error with bad request"},
 			expCode:  codes.InvalidArgument,
 		},
+		{
+			name: "googleapi.Error that wraps apierror.APIError, which wraps a googleapi.Error",
+			inputErr: getAPIWrappedError(&googleapi.Error{
+				Code:    403,
+				Message: "Some permission error",
+			}, true /* wrap */),
+			expCode: codes.PermissionDenied,
+		},
+		{
+			name: "googleapi.Error that wraps apierror.APIError of http kind",
+			inputErr: getAPIWrappedError(&googleapi.Error{
+				Code:    404,
+				Message: "data requested not found error",
+			}, false /* wrap */),
+			expCode: codes.NotFound,
+		},
```

Comment:
Isn't googleapi.Error the same as http kind? If so, can we change http kind to googleapi.Error to be consistent. If they are the same, then I think this is the same test case as the one above.